src/test/java/com/codurance/lightaccess/PGOneToManyShould.java

```diff
@@ -0,0 +1,82 @@
+package com.codurance.lightaccess;
+
+import org.junit.Test;
+import java.util.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
+public class PGOneToManyShould {
+
+    private static final int SOME_KEY = 1;

```

Comment:
I would use some string instead of integers. You can give more meaningful values for such a kind of variables.